conf/fluent-bit.conf

```diff
@@ -45,4 +45,4 @@
 
 [OUTPUT]
     Name  stdout
-    Match *
```

Comment:
This diff is due to "\n" at the end of the file has been trimmed.

Generally not having a newline at the end is frowned upon especially
in C projects, so you would be better not to trim the character here.

Reply:
@aSquare14 It does not seem to be fixed. If you cannot make this
straight using your editor, try the following:

```bash
$ git checkout master conf/fluent-bit.conf
$ git add conf/fluent-bit.conf
$ git commit --amend
```